addons/mail/data/discuss_channel_data.xml

```diff
@@ -14,6 +14,13 @@
             <field name="group_ids" eval="[Command.link(ref('base.group_system'))]"/>
         </record>
 
+        <record model="discuss.channel" id="mail.channel_public">
```

Comment:
This file is data not demo, it shouldn't be there (see also other comments about file structure).